### ISSUE TITLE
Remove sensor from Node Red should also remove from HA

### ISSUE THREAD

**Issue by declanshanaghy** [zachowj/hass-node-red#10]
**Is your feature request related to a problem? Please describe.**
When i deleted a sensor from Node Red is was not deleted from the integration. I created another one with the same name but now there are 2 sensors in the integration and the second one is suffixed with "_2". Hence I now seem to have lost control of the first one.

**Describe the solution you'd like**
Deleting the sensor from node red should delete it from the integration into HA

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by zachowj** [comment]
Yes, sensors and switches should automatically be removed from HA when they are removed from NR, but only if there is a current connection from NR to HA when you deploy those changes.

Documentation is definitely lacking here most of it is in the NR package release notes or in the NR docs. The switches can be created from any of the event nodes, events:all, events:state, trigger:state, and poll state nodes, using the expose `Expose to Home Assistant`

<https://community.home-assistant.io/t/node-red-contrib-home-assistant-websocket/71032/389?u=kermit>

**Comment by declanshanaghy** [comment]
To verify entity deletion i made a test switch entity and deployed it. The entity shows up and I can toggle it from HA. I then deleted the entity from the node red flow. The entity then changes to unavailable state. It doesn't get removed. There was a valid connection from NR to HA when I deployed the change with the entity deleted.

**Comment by dibyadip-das** [comment]
The sensor node is removed in NR after installing this repo using HACS. Still, I am confused as to how I can add new sensors?
